docker/README.md

```diff
@@ -0,0 +1,98 @@
+**This documentation focuses on the Deployer Docker image and its usage, not the Deployer itself. If you want learn more about the Deployer, please read [its documentation](../packages/deployer/README.md).**
+
+# Build
+You can build the Docker image by running:
+```bash
+docker build -f docker/Dockerfile -t api3/deployer:latest .
+```
+
+Before the build, all Deployer dependencies (but not the Deployer itself) must be available in the package registry.
+
+> You can use build argument `yarnRegistry` to specify a different package registry for package installation. This is useful for testing a Docker build with unreleased dependencies. You can use for example [Verdaccio](https://verdaccio.org/) for this purpose.
+
+# Configuration
+## Credentials
+In order to deploy Airnode to cloud you need to pass your cloud credentials to the container. We're currently supporting only AWS cloud.
+
+### AWS
+You have two options how to pass your AWS credentials
+
+#### Via environment variables:
+You can either prepare a file with environment variables and pass that
+```bash
+$ cat aws.env
```

Comment:
The user might not know what `cat` does. I would say the instructions should rather say something along the lines of "create a `aws.env` file with the following contents: ..."